docs/configuration.md

```diff
@@ -117,9 +117,9 @@ Settings for the dashboard [reporter](#reporters-string). See the [dashboard doc
 **Since:** `v0.10.0`  
 **Description:**
 
-See [timeout](#timeout-number)
+See [timeout](#timeout-duration)
```

Comment:
You Scala devs think you soooo smart!! 🤓

Reply:
![](https://media1.tenor.com/images/11605b822e835ac00362e2fa05c5256a/tenor.gif)